pages/ow/fish/howto/index.vue

```diff
@@ -270,7 +270,7 @@
               an insane amount of standing and resources that you will need for later on. To do so, you need the
               appropriate bait to maximize the chances for those fish to spawn. Rare fishes do spawn in Orb Vallis
               without their respective bait, but definitely not as often compared to when you do use their bait. Again,
-              you can refer to the <nuxt-link to="vallis-fish">Super Fishing Reference Table</nuxt-link> for its
+              you can refer to the <nuxt-link to="vallis/fish">Super Fishing Reference Table</nuxt-link> for its
```

Comment:
For some reason this and all other links I have tested still redirects to `ow/fish/poe-fish` instead of `poe/fish`

Reply:
might need to purge locally :thinking: i'll try out the vercel deploy

Reply:
oh, looks like there's like 4 of the exact same link

Reply:
@EricSihaoLin it's looking right for me now if you open the latest deployment in an incognito tab